WordPress/Classes/Services/PostService.m

```diff
@@ -492,6 +492,10 @@ - (void)restorePost:(AbstractPost *)post
 - (void)initializeDraft:(AbstractPost *)post {
     post.remoteStatus = AbstractPostRemoteStatusLocal;
     post.status = PostStatusPublish;
+
+    // Hack: The date_create_gmt should arleady be nil for a draft but
```

Comment:
Could you attach your name and a date to this? e.g. `HACK(aerych, 6/18/2015):...`